recipes/vnu-validator/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+noarch_platform:
+  - unix  # [osx]
+  - unix  # [linux]
+  - win  # [win]
```

Comment:
```suggestion
  - linux_64
  - win_64
```

Reply:
thanks for the review!,

we've got that over in `conda-forge.yml`: here, it's being used as the name of `__{virtual platform}`, as we nly want to end up with 2 builds (`unix` and `win`).

This is adapted from `ipython-feedstock`:
- https://github.com/conda-forge/ipython-feedstock/blob/main/recipe/conda_build_config.yaml
- https://github.com/conda-forge/ipython-feedstock/blob/main/conda-forge.yml#L18

Reply:
That's overly complicated. Here's a simplification of the ipython-feedstock. https://github.com/conda-forge/ipython-feedstock/pull/205

Reply:
yeah, tried that locally, getting:

```
recipes/vnu-validator/ has some lint:
  `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: generic`.
```